msfinstall

```diff
@@ -0,0 +1,156 @@
+#!/bin/sh
+
+print_pgp_key() {
```

Comment:
Why is this file included? Can we remove this?

Reply:
yeap remove this

Reply:
confirmed